msg/DetectedObject.msg

```diff
@@ -1,6 +1,6 @@
-int32 label
+string label
 float32 score
-float32 left
-float32 right
-float32 top
-float32 bottom
\ No newline at end of file
+int32 left
+int32 top
+int32 right
+int32 bottom
```

Comment:
I think it's better to use floating point (`0`-`1`, represent position percentage on image) instead of integers for coordinate.